cert-manager/cert-manager
# Fix trivy vulns

### Pull Request Motivation



Seems like v0.38.0 of otelhttp package imported by cert-manager master has a vulnerability https://prow.build-infra.jetstack.net/view/gs/jetstack-logs/logs/ci-cert-manager-master-trivy-test-controller/1655574164670517248 that has been fixed in v0.39.0.

To reproduce the issue:
- Run `make trivy-scan-all` from current master
- Oberve a failure

To verify fix:
- Run `make trivy-scan-all` from  this PR
- Observe success

cert-manager has never been released with this version of otelhttp

(cert-manager release-1.11 and release-1.10 trivy tests are currently also failing, but because of a Docker vulnerability)

### Release Note



```release-note
NONE
```
